lib/aggregators/top-balances.ts

```diff
@@ -1,25 +1,29 @@
 import * as c32check from 'c32check';
-import Aggregator from './aggregator';
+import { AggregatorWithArgs } from './aggregator';
 import { getUnlockedSupply, getTopBalances } from '../core-db-pg/queries';
 import { microStacksToStacks } from '../utils';
 
-export interface TopBalanceAccount {
+export type TopBalanceAccount = {
```

Comment:
A hotly contested TypeScript style. 

Is there a reason to use a type literal over an `interface`? I've used [this rule in the past](https://palantir.github.io/tslint/rules/interface-over-type-literal/). Saving `type`s for unions.

I'm wondering if a shared eslint-config enforcing rules like this and `Array<any>` vs `any[]` is a good idea, or whether the conversations it'll cause will outweigh the benefits of consistency. Thoughts?

Reply:
I know right :(
I've previously refactored all `type` declarations to `interface` in another codebase, following some recommended eslint rules. 

In this case, almost all these type defs are used with json serialization, which I have enforced using that `Json` type (recently working correctly in typescript):
```ts
export type Json =
    | string
    | number
    | boolean
    | null
    | { [property: string]: Json }
    | Json[];
```
When I last checked, some quirks related `interface` indexer behavior caused seemingly valid `interface`s to not work with the `Json` type. 

Just tested it again and seeing same issue:
![image](https://user-images.githubusercontent.com/1447546/71989629-fed4c580-31ff-11ea-9823-38c73fc67b6a.png)

Reply:
IIUC, if this feature is merged, it would at least allow work-arounds for making `interface` types to match that `Json` type https://github.com/microsoft/TypeScript/pull/26797

Other than that, I haven't figured out any other solutions. 

To your question about the shared style rules -- I personally prefer `any[]` and `interface` (when possible), but I think the stylistic inconsistency between them is minuscule and probably not worth having rules for.